.github/workflows/code-formatting.yml

```diff
@@ -62,7 +62,7 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v3
         with:
-          ref: ${{ github.head_ref }}
+          ref: ${{ github.ref }}
```

Comment:
I might be late to the party, but you could even just remove the ref argument.

Reply:
We need this reference. I had some issues with the checkout made to a branch I did not expect

Reply:
As far as I remember, without the option, it checks out the expected merge version, which is what you need to verify. I assume the problem you had was about tagging.